crates/router/src/connector/fiserv/transformers.rs

```diff
@@ -18,11 +19,18 @@ pub struct FiservPaymentsRequest {
     transaction_interaction: TransactionInteraction,
 }
 
-#[derive(Default, Debug, Serialize, Eq, PartialEq)]
-#[serde(rename_all = "camelCase")]
-pub struct Source {
-    source_type: String,
-    card: CardData,
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(tag = "sourceType")]
+pub enum Source {
+    PaymentCard {
+        card: CardData,
+    },
+    #[allow(dead_code)]
+    GooglePay {
+        data: String,
+        signature: String,
+        version: String,
+    },
```

Comment:
consider removing the `#[allow(dead_code)]` or the `GooglePay` varient, you can do this is a separate PR if need be.